web-services/model/src/main/java/datawave/webservice/query/model/ModelBean.java

```diff
@@ -326,70 +305,160 @@ public VoidResponse cloneModel(@Required("name") @FormParam("name") String name,
     @Path("/{name}")
     @GZIP
     @Interceptors({RequiredInterceptor.class, ResponseInterceptor.class})
-    public datawave.webservice.model.Model getModel(@Required("name") @PathParam("name") String name, @QueryParam("modelTableName") String modelTableName) {
-        
-        if (modelTableName == null) {
-            modelTableName = defaultModelTableName;
-        }
-        
-        datawave.webservice.model.Model response = new datawave.webservice.model.Model(jqueryUri, dataTablesUri);
-        
-        // Find out who/what called this method
-        Principal p = ctx.getCallerPrincipal();
-        String user = p.getName();
-        Set<Authorizations> cbAuths = new HashSet<>();
-        if (p instanceof DatawavePrincipal) {
-            DatawavePrincipal cp = (DatawavePrincipal) p;
-            user = cp.getShortName();
-            for (Collection<String> auths : cp.getAuthorizations()) {
-                cbAuths.add(new Authorizations(auths.toArray(new String[auths.size()])));
-            }
-        }
-        log.trace(user + " has authorizations " + cbAuths);
+    public Model getModel(@Required("name") @PathParam("name") String name, @QueryParam("modelTableName") String tableName,
```

Comment:
It is imperative that we still support the previous interface.  So all additional parameters need to have defaults or we need to create a completely separate endpoint.